docs/ManualDeploymentGuideScalarDLOnAzure-1.md

```diff
@@ -0,0 +1,217 @@
+
+## Step 1. Configure your network
+
+Configure a secure network with your organizational or application standards.
+Scalar DL handles highly sensitive data of your application, so you should create a highly secure network for production.
+This section shows how to configure a secure network for Scalar DL deployments.
+
+### Requirements
+
+* You must create a virtual network with a subnet for bastion.
+* You must create 2 subnets with the prefix of at least `/22` for AKS, one subnet must be created with the name `k8s_ingress` to create an envoy load balancer.
+
+### Recommendations
+
+* You should create a bastion server to manage the Kubernetes cluster for production.
+
+### Steps
+
+* Create an Azure virtual network on the basis of [Azure official guide](https://docs.microsoft.com/en-us/azure/virtual-network/quick-create-portal) with the above requirements and recommendations.
+* Create a bastion server on the basis of [Azure official guide](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-portal).
+
+## Step 2. Set up a database
+
+In this section, you will set up a database for Scalar DL.
+
+### Requirements
+
+* You must have a database that Scalar DL supports.
+
+### Steps
+
+* Follow [Set up a database guide](./SetupDatabase.md) to set up a database for Scalar DL.
+
+## Step 3. Configure AKS
+
+This section shows how to configure a Kubernetes service for Scalar DL deployment.
+
+### Prerequisites
+
+Install the following tools on the bastion for controlling the AKS cluster:
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli): In this guide, Azure CLI is used to create a kubeconfig file to access the AKS cluster.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/): Kubernetes command-line tool to manage AKS cluster. Kubectl 1.19 or higher is required.
+
+### Requirements
+
+* You must have an AKS cluster with Kubernetes version **1.19** or higher for Scalar DL deployment.
+* You must create a new `user node pool` with the name `scalardlpool` for Scalar DL deployment.
+* You must select the subnet other than `k8s_ingress` subnet for creating the AKS cluster.
+* You must create a Kubernetes cluster with `service principal` as the Authentication method.
+* You must create a Kubernetes cluster with `Azure CNI`.
+* You must add a role assignment to the `k8s_ingress` subnet to access the newly created AKS cluster service principal using the role of `Network Contributor`.
+
+### Recommendations
+
+* You should use Kubernetes node size `Standard D2s v3` for Scalar DL node pool.
+* You should create 3 nodes in each node group for high availability in production.
+* You should configure autoscaling for the AKS cluster on the basis of [Azure official guide](https://docs.microsoft.com/en-us/azure/aks/cluster-autoscaler#set-the-cluster-autoscaler-profile-on-an-existing-aks-cluster).
```

Comment:
I think we can enable `Autoscale` when setting up `nodepool`, but do we need this too?

Reply:
Thank you, We dont need to do this, removed the recommendation in b2c656c.

Reply:
I think we need to specify the auto scaling option in recommendations because we have a section for [Maintain Scalar DL pods](https://github.com/scalar-labs/scalar-kubernetes/blob/manual-guide-azure/docs/MaintainPods-1.md)